downstream/modules/platform/ref-operator-ocp-version.adoc

```diff
@@ -4,8 +4,6 @@
 
 [role="_abstract"]
 
-The {OperatorPlatform} to install {PlatformNameShort} {PlatformVers} is available on {OCPShort} 4.9 and later versions.
+The {OperatorPlatform} to install {PlatformNameShort} {PlatformVers} is available on {OCPShort} 4.10 to 4.13.
```

Comment:
Hey @annibond, it's possible that I'm just confused, but I don't see any of the title changes you mentioned in your comment in the Conversation tab. Could you tell me how this change leads to the title changes? Apologies if I'm misunderstanding.